fhir_2_ada-r4/mp/9.3.0/payload/2.0.0-beta.3/mp-MedicationAdministration.xsl

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright © Nictiz
+
+This program is free software; you can redistribute it and/or modify it under the terms of the
+GNU Lesser General Public License as published by the Free Software Foundation; either version
+2.1 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Lesser General Public License for more details.
+
+The full text of the license is available at http://www.gnu.org/copyleft/lesser.html
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:f="http://hl7.org/fhir" xmlns:local="urn:fhir:stu3:functions" xmlns:nf="http://www.nictiz.nl/functions" xmlns:util="urn:hl7:utilities" exclude-result-prefixes="#all" version="2.0">
+
+	<xsl:variable name="urlMpMedicationAdministration"><xsl:value-of select="$urlBaseNictizProfile"/>mp-MedicationAdministration</xsl:variable>
+
+	<xd:doc>
+		<xd:desc>Template to convert f:MedicationAdministration to ADA medicatietoediening</xd:desc>
+	</xd:doc>
+	<xsl:template match="f:MedicationAdministration" mode="mp-MedicationAdministration">
+		<medicatietoediening>
+			<!-- identificatie -->
+			<xsl:apply-templates select="f:identifier" mode="#current"/>
+			<!-- toedienings_product -->
+			<xsl:apply-templates select="f:medicationReference" mode="#current"/>
+			<!-- toedienings_datum_tijd -->
+			<xsl:apply-templates select="f:effectiveDateTime" mode="#current"/>
+			<!-- afgesproken_datum_tijd -->
+			<!-- MP-1408 LR: afgesproken_datum_tijd nolonger supported from MP 9.3 beta.3 onwards but kept in stylesheet due to backwards compatibility-->
+			<xsl:apply-templates select="f:extension[@url = $urlExtMedicationAdministration2AgreedDateTime]" mode="#current"/>
+			<!-- geannuleerd_indicator -->
+			<xsl:if test="f:status/@value = 'entered-in-error'">
+				<geannuleerd_indicator value="true"/>
+			</xsl:if>
+			<!-- toegediende_hoeveelheid -->
+			<xsl:apply-templates select="f:dosage/f:dose" mode="#current"/>
+			<!-- afgesproken_hoeveelheid -->
+			<!-- MP-1395 LR: afgesproken_hoeveelheid removed -->
```

Comment:
You might consider to just remove line 39 and 40 completely (since the difference already follows from the stylesheet in the 2.0.0-beta.1 folder).